index.html

```diff
@@ -696,23 +780,23 @@ <h2>
         <ol>
           <li>If the screen width is greater than the screen height, set the
           <a>document</a>'s <a>current orientation type</a> to
-          <dfn>landscape-primary</dfn> or <dfn>landscape-secondary</dfn>.
+          <a>landscape-primary</a> or <a>landscape-secondary</a>.
           </li>
           <li>Otherwise, if the screen width is less than or equal to the
           screen height, set the <a>document</a>'s <a>current orientation
-          type</a> to <dfn>portrait-primary</dfn> or
-          <dfn>portrait-secondary</dfn>.
+          type</a> to <a>portrait-primary</a> or <a>portrait-secondary</a>.
           </li>
           <li>Set the <a>document</a>'s <a>current orientation angle</a> to the
           clockwise angle in degrees between the orientation of the viewport as
-          it is drawn and the natural orientation of the device (i.e., the top
-          of the physical screen). This is the opposite of the physical
+          it is drawn and the <a>natural</a>orientation of the device (i.e.,
+          the top of the physical screen). This is the opposite of the physical
           rotation. In other words, if a device is turned 90 degrees on the
           right, the <a>current orientation angle</a> would be 270 degrees.
           </li>
         </ol>
       </section>
-      <section data-dfn-for="OrientationLockType">
+      <section data-dfn-for="OrientationLockType" data-link-for=
```

Comment:
This shouldn't define `OrientationLockType` things... if it does, then that's a bug.  

```suggestion
      <section data-link-for=
```